models/autoencoders/discreteVAE.py

```diff
@@ -11,7 +11,7 @@
 from models.networks.basicCoders import BasicDecoder
 from models.networks.hierarchicalEncoder import HierarchicalEncoder
 from models.rbm.rbm import RBM
-from models.samplers.contrastiveDivergence import ContrastiveDivergence
+#from models.samplers.contrastiveDivergence import ContrastiveDivergence
```

Comment:
Commenting this out will the ContrastiveDivergence module inaccessible further below.

Reply:
@abhishekabhishek , I added this back in with commit 8f9e3bcf5935019aaeaf873656c18f63cdc3fd85

Reply:
Sounds good. This was generating a weird "Tab Indentation Error" for many lines of the module so I commented importing the module to prevent that and continue with the development. We can discuss more on this since this error is happening a lot more recently.